scss/components/_control-bar.scss

```diff
@@ -97,6 +97,10 @@
       cursor: auto;
     }
 
+    .oo-control-bar-item.oo-quality, .oo-control-bar-item.oo-closed-caption{
```

Comment:
Let's make sure that there isn't another button remaining that triggers the control bar misalignment. It might be better to use a more generic approach instead of applying the style to individual elements.